target/linux/ramips/mt7620/base-files/etc/board.d/01_leds

```diff
@@ -81,6 +81,12 @@ dlink,dwr-960)
 	ucidef_set_led_switch "lan" "lan" "green:lan" "switch0" "0x2e"
 	ucidef_set_led_switch "wan" "wan" "green:wan" "switch0" "0x01"
 	;;
+domywifi,dw22d)
+	ucidef_set_led_netdev "wlan2g" "wlan2g" "green:wlan2g" "wlan1"
+	ucidef_set_led_netdev "wlan5g" "wlan5g" "green:wlan5g" "wlan0"
+	ucidef_set_led_netdev "wan_link" "wan_link" "red:internet" "eth0.2" "link"
+	ucidef_set_led_netdev "wan_txrx" "wan_txrx" "green:internet" "eth0.2" "tx rx"
```

Comment:
Is this vendor behavior or your invention?

Reply:
The vendor setting is when the network is connected，show green led otherwise show red. It's not easy to achieve it so I just drop it now and user can customize it according to their preferences.

Reply:
Well, just use the green:internet for "link tx rx" as everyone else does in this case. And please use the proper switch-based setup for that.